sdk/metricsadvisor/ai-metrics-advisor/src/transforms.ts

```diff
@@ -931,8 +931,8 @@ export function fromServiceDataFeedDetailUnion(
         ...common,
         source: {
           dataSourceType: "AzureApplicationInsights",
-          azureCloud: orig.dataSourceParameter.azureCloud,
-          applicationId: orig.dataSourceParameter.applicationId,
+          azureCloud: orig.dataSourceParameter.azureCloud!,
+          applicationId: orig.dataSourceParameter.applicationId!,
```

Comment:
Is `orig` coming from the service? How do we know that it's safe to use the non-null assertion?